website/docs/components/form/masked-input/index.md

```diff
@@ -9,6 +9,8 @@ related: ['components/form/text-input', 'components/form/textarea']
 previewImage: assets/illustrations/components/form/masked-input.jpg
 navigation:
   keywords: ['masked' ,'sensitive', 'obscured', 'obfuscated', 'secret', 'form', 'input', 'textarea']
+status:
+  updated: 4.11.1
```

Comment:
```suggestion
  updated: 4.12.0
```

Reply:
I made the changes locally & pushed up. Thanks